website/sidebars.js

```diff
@@ -437,30 +437,34 @@ const sidebars = {
         },
         {
           type: 'doc',
-          id: 'stylus/concepts/stylus-cache-manager',
-          label: 'Stylus caching strategy',
-        },
-        {
-          type: 'html',
-          value:
-            '<a class="menu__link menu__list-item" href="/run-arbitrum-node/run-local-dev-node">Run a Stylus dev node<span class="other-section-icon">↑</span></a>',
-          // q: why use an anchor html tag here?/node-running/how-tos/running-an-stylus-node
-          // a: see note at end of file
-        },
-        {
-          type: 'doc',
-          label: 'Testnets',
+          label: 'Testnet',
           id: 'stylus/reference/testnet-information',
         },
         {
           type: 'link',
           label: 'Stylus by example',
           href: 'https://stylus-by-example.org/',
         },
-
         {
           type: 'category',
-          label: 'Gas and ink',
+          label: 'Stylus Rust SDK',
+          collapsed: true,
+          items: [
+            {
+              type: 'doc',
+              id: 'stylus/reference/rust-sdk-guide',
+              label: 'Rust SDK overview',
+            },
+            {
+              type: 'link',
+              label: 'Rust crate docs',
+              href: 'https://docs.rs/stylus-sdk/latest/stylus_sdk/index.html',
+            },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'Gas, ink and caching',
```

Comment:
Should we replace ink with Ink for the sake of consistency?

Reply:
I see that I haven't used that in other places though. I'll update them.

Reply:
Updated, thanks for the flag!